src/flexmock/_api.py

```diff
@@ -562,32 +565,20 @@ def __init__(
         self._verified = False
         self._callable = True
         self._local_override = False
+        # Remove in 1.0
```

Comment:
Can you make an issue for release 1.0 and put the #number here, so it's easier to find ? we might end up with a couple more of those.

Reply:
Done! Issue #96. I also made a milestone for 1.0 version.